inference_lib/setup.cfg

```diff
@@ -0,0 +1,50 @@
+[metadata]
+name = aqlm
+version = 1.0.0dev9
+author = Vage Egiazarian, Andrei Panferov and Elias Frantar
+author_email = vahe527887@yandex.ru
+description = Efficiently run models quantized with AQLM
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Vage1994/AQLM
+project_urls =
+    Bug Tracker = https://github.com/Vage1994/AQLM/issues
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    Intended Audience :: Science/Research
+    License :: OSI Approved :: MIT License
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Mathematics
+    Topic :: Scientific/Engineering :: Artificial Intelligence
+    Topic :: Software Development
+    Topic :: Software Development :: Libraries
+    Topic :: Software Development :: Libraries :: Python Modules
+
+[options]
+package_dir =
+    = src
+packages = find:
+include_package_data = True
+python_requires = >=3.10
+install_requires =
+    torch>=2.1
+    transformers>=4.37
```

Comment:
Nit/recommendation: upper bound transformer version, e.g. >=0.37,<0.38

Why: your benchmarking code relies on HF internals that often break with major patches

Reply:
Can't make it work. The version range isn't properly parsed for some reason